src/pages/SignIn/SignIn.jsx

```diff
@@ -1,55 +1,105 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import { NavBar } from "../HomePage/HomePageComponents";
+import { Modal } from "../Modal";
+import { UserInfo } from "../UserInfo/UserInfo";
 import SignInCSS from "../SignIn/SignIn.module.css";
+import { useAuth } from "../../contexts/authContext";
+import { GlobalCartContext } from "../../contexts/cartContext";
+import { useState } from "react";
 function SignIn() {
+  const { loginUser, userDispatch, isLoggedIn } = useAuth();
+  const { isModalOpen, dispatch } = GlobalCartContext();
+  setTimeout(() => {
+    dispatch({ type: "REMOVE_MODAL" });
+  }, 10000);
+  const [formData, setFormData] = useState({
+    email: "",
+    password: "",
+  });
   return (
     <div>
       <NavBar />
-      <div className={SignInCSS["login-container"]}>
-        <h1 className={SignInCSS["header-large"]}>LOG IN</h1>
-        <div className={SignInCSS["input"]}>
-          <label htmlFor="input-box">E-mail adddress</label>
-          <input
-            className={`input-box basic ${SignInCSS["text-field"]}`}
-            type="text"
-            placeholder="user@yahoo.com"
-          />
-        </div>
-        <div className={SignInCSS["input"]}>
-          <label htmlFor="input-box">Password</label>
-          <input
-            className={`input-box basic ${SignInCSS["text-field"]}`}
-            type="password"
-            placeholder="************"
-          />
-        </div>
-
-        <div className={SignInCSS["handy-div"]}>
-          <div className={SignInCSS["checkbox-text"]}>
-            <input className={SignInCSS["check-box"]} type="checkbox" />
-            <label className={SignInCSS["check-label"]} htmlFor="check-box">
-              Remind Me
-            </label>
+      {isLoggedIn ? (
+        <UserInfo />
+      ) : (
+        <div className={SignInCSS["login-container"]}>
+          <h1 className={SignInCSS["header-large"]}>LOG IN</h1>
+          <div className={SignInCSS["input"]}>
+            <label htmlFor="input-box">E-mail adddress</label>
+            <input
+              className={`input-box basic ${SignInCSS["text-field"]}`}
+              type="text"
+              placeholder="user@yahoo.com"
+              value={formData.email}
+              onChange={(e) =>
+                setFormData((prev) => ({ ...prev, email: e.target.value }))
+              }
+            />
+          </div>
+          <div className={SignInCSS["input"]}>
+            <label htmlFor="input-box">Password</label>
+            <input
+              className={`input-box basic ${SignInCSS["text-field"]}`}
+              type="password"
+              placeholder="************"
```

Comment:
in the placeholder type password, as when a person with disability uses it it will say this is the field for password